### DISCUSSION TITLE
ECW file support for templates

### DISCUSSION THREAD

**Discussion by M0OJM** [OpenOrienteering/mapper#1910] [Ideas]
ECW files are ideal for satellite imagery such a Google Earth and they are much smaller than tiffs.  It appears GDAL supports this format but they do not seem visible when I try to use them as a template in OOM. Am I missing a trick?

**Comment by dg0yt** [comment]
GDAL can be used together with a commercial software development kit (SDK) which adds the actual ECW support. But this SDK is a closed-source commercial product. It cannot be combined with open source software under GPL3 license.
